spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -178,6 +178,96 @@ public void testSave() throws Exception {
 	}
 
 	@Test
+	public void testFindRelatedStreams() throws Exception {
```

Comment:
This specific method doesn't look like a business logic IMO as it more looks like search/filter changes at the `controller` side.